apprise/plugins/NotifyPushover.py

```diff
@@ -240,6 +251,24 @@ def __init__(self, token, targets=None, priority=None, sound=None,
         else:
             self.priority = priority
 
+        # The following are for emergency alerts
+        if self.priority == PushoverPriority.EMERGENCY:
+            if retry is None or expire is None:
+                msg = 'Emergency requires expire and retry to be specified.'
+                self.logger.warning(msg)
+                raise TypeError(msg)
+            if retry < 30:
+                msg = 'Retry must be at least 30.'
+                self.logger.warning(msg)
+                raise TypeError(msg)
+            if expire < 0 or expire > 10800:
+                msg = 'Expire has a max value of at most 10800 seconds.'
+                self.logger.warning(msg)
+                raise TypeError(msg)
+
+            self.retry = retry
+            self.expire = expire
```

Comment:
This section here just needs a wee bit of clean-up; but what you have is so close to being perfect. Have a look at the [NotifyNexmo](https://github.com/caronc/apprise/blob/master/apprise/plugins/NotifyNexmo.py) plugin and the **__init__()** function.

It handles *ints* and additionally provides some default values if none are specified.  I think you should do the same here.  It will mean the following though:
* 2 new tests that set **retry=invalid** and **expire=invalid** (in _test_rest_plugins.py_ - just 2 more URLs)
* Set the default values if the *EMERGENCY* _priority_ is set and either _retry_ and/or expire isn't set. I trust you on what these defaults should be. We can change them later someone files a bug report.  Pick something pretty that you'd expect it to be.
* with respect to your error checking above, that's all good though; you do want to throw errors if retry and/or expires is NOT none and they're defined outside the boundaries. So good job here.

Reply:
I was on the fence if I should create default values for retry and expire, I will add that! Thank you for the review~